sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Azure.AI.Language.Conversations.csproj

```diff
@@ -5,6 +5,7 @@
     <Version>1.0.0-beta.3</Version>
     <PackageTags>Azure AI Language Conversations LUIS</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
```

Comment:
Magic variables seem a lot harder to remember / learn. We already likely included a bunch of extra files like you're removing below so why make the process different? That just seems more complicated, especially for anyone onboarding to our code. Why can't new projects just do the same, or even a generator that might produce the .csproj add those 6 lines of XML to include the 4 files you need?

Reply:
Per offline conversation, the idea was to make it easier to refactor LRO source changes without subsequent 50+ project updates every time.

Reply:
The advantage here is that its one magic variable versus having to know a variable number of magic variables which are the list of files I need for LRO.  In this way you can simply say, give me the LRO files as one line and it doesn't matter what the list of files are.

Also we will be making changes to this list of files and being able to change it in one place is better than having to change 56 csproj files every time.